csharp/src/Apache.Arrow.Adbc/C/CAdbcDriverExporter.cs

```diff
@@ -127,12 +127,13 @@ public unsafe static AdbcStatusCode AdbcDriverInit(int version, CAdbcDriver* nat
             nativeDriver->StatementBindStream = StatementBindStreamPtr;
             nativeDriver->StatementExecuteQuery = StatementExecuteQueryPtr;
             nativeDriver->StatementExecutePartitions = StatementExecutePartitionsPtr;
-            nativeDriver->StatementGetParameterSchema = StatementGetParameterSchemaPtr;
+            nativeDriver->StatementExecuteSchema = StatementExecuteSchemaPtr;
             nativeDriver->StatementNew = StatementNewPtr;
             nativeDriver->StatementPrepare = StatementPreparePtr;
             nativeDriver->StatementRelease = StatementReleasePtr;
             nativeDriver->StatementSetSqlQuery = StatementSetSqlQueryPtr;
             nativeDriver->StatementSetSubstraitPlan = StatementSetSubstraitPlanPtr;
+            nativeDriver->StatementGetParameterSchema = StatementGetParameterSchemaPtr;
```

Comment:
These fields are deliberately initialized in the order that they are declared in `adbc.h` to make it easy to compare the two. If you look at that definition, you'll see that `StatementGetParameterSchema` does indeed come immediately after `StatementExecutePartitions` and that `StatementExecuteSchema` doesn't come until the 1.1 section.

Reply:
Ahhh, I see, thank Curt for the clarification.